packages/core/contracts/oracle/implementation/DesignatedVotingV2.sol

```diff
@@ -3,8 +3,9 @@ pragma solidity ^0.8.0;
 
 import "../../common/implementation/MultiRole.sol";
 import "../../common/implementation/Stakeable.sol";
-import "../interfaces/VotingAncillaryInterface.sol";
+import "../interfaces/VotingV2Interface.sol";
```

Comment:
other option that might make sense is to add the `multicall` method to the voting contract interface within the `[VotingV2Interface.sol` file rather than importing MultiCaller again. wdyt? this will also  mean you don't need to do any casting.

Reply:
I think if we drop the batch methods and have this contract be multicaller, too, that will naturally eliminate this problem.